source/server/server.cc

```diff
@@ -60,6 +60,7 @@
 #include "source/server/listener_manager_factory.h"
 #include "source/server/regex_engine.h"
 #include "source/server/ssl_context_manager.h"
+#include "source/server/utils.h"
```

Comment:
you'll also want to remove the serverState function in the anonymous namespace of this file

Reply:
Ah whoops, I meant to do that before pushing.